BepInExTests/BepInExTests.csproj

```diff
@@ -39,10 +39,4 @@
     <ProjectReference Include="..\BepInEx.IL2CPP\BepInEx.IL2CPP.csproj" />
     <ProjectReference Include="..\BepInEx.Unity\BepInEx.Unity.csproj" />
   </ItemGroup>
-  <ItemGroup>
-    <Folder Include="Properties" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Remove="Properties\AssemblyInfo.cs" />
-  </ItemGroup>
```

Comment:
How is this related to semver change? This should be in a separate PR for better testability and ability to revert it if needed.